Added some documentation for encrypting with age (#1001)
getsops/sops: README.rst

Encrypting with SSH keys via age is not yet supported by sops.

> A list of age recipients can be added to the .sops.yaml:

Reviewer: ```suggestion
A list of age recipients can be added to the ``.sops.yaml``:
```

.. code:: yaml

    creation_rules:
        - age: >-
            age1s3cqcks5genc6ru8chl0hkkd04zmxvczsvdxq99ekffe4gmvjpzsedk23c,
            age1qe5lxzzeppw5k79vxn3872272sgy224g2nzqlzy3uljs84say3yqgvd0sw

> It is also possible to use updatekeys, when adding or removing age recipients.
> 
> I.e. :

Reviewer: ```suggestion
It is also possible to use ``updatekeys``, when adding or removing age recipients. For example:
```
